core/sql/SqlCompilerDebugger/ExeQueueDisplay.h

```diff
@@ -33,22 +18,43 @@
 // under the License.
 //
 // @@@ END COPYRIGHT @@@
- *
- *
- *****************************************************************************
- */
+#ifndef EXEQUEUEDISPLAY_H
```

Comment:
I have no clue why git says I renamed ComSqlexedbg.h to ExeQueueDisplay.h. In reality, I deleted file ComSqlexedbg.h and added a new file ExeQueueDisplay.h. Please ignore the the ComSqlexedbg.h file, it doesn't have any meaningful content.